client/blocks/plan-purchase-features/index.jsx

```diff
@@ -29,47 +29,45 @@ import MonetizeSite from './monetize-site';
 import CustomDomain from './custom-domain';
 import GoogleAnalyticsStats from './google-analytics-stats';
 import HappinessSupport from './happiness-support';
-import CurrentPlanHeader from './current-plan-header';
 import JetpackAntiSpam from './jetpack-anti-spam';
 import JetpackBackupSecurity from './jetpack-backup-security';
 import JetpackReturnToDashboard from './jetpack-return-to-dashboard';
 import JetpackSurveysPolls from './jetpack-surveys-polls';
 import JetpackWordPressCom from './jetpack-wordpress-com';
-import { getPlansBySite } from 'state/sites/plans/selectors';
-import { getSelectedSite } from 'state/ui/selectors';
+import CurrentPlanHeader from './current-plan-header';
 import {
 	isPremium as isWpcomPremium,
 	isBusiness as isWpcomBusiness
 } from 'lib/products-values';
 import { isWordadsInstantActivationEligible } from 'lib/ads/utils';
+import { plansList } from 'lib/plans/constants';
 
 class PlanPurchaseFeatures extends Component {
 	static propTypes = {
 		plan: PropTypes
 			.oneOf( [ PLAN_FREE, PLAN_PERSONAL, PLAN_PREMIUM, PLAN_BUSINESS ] )
 			.isRequired,
-		selectedSite: PropTypes.object
+		selectedSite: PropTypes.object,
+		sitePlans: PropTypes.object,
+		page: PropTypes
```

Comment:
The way I designed it, we can do anything differently based on page we are on. For example, `current-plan` need a different header from `checkout-thank-you` page. But what if checkout-thank-you page also needs a footer? Well, as said in https://github.com/Automattic/wp-calypso/pull/7538#discussion_r75523206, we could just pass the header and footer as children to `plan-purchase-features`, but what if we need something else than those two?